change/@azure-msal-node-c75d4701-e734-4cf0-8cad-7953484e6703.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
Looks like there's changes to node, please update to the relevant change type

Reply:
I have changed it to a "minor" change